packages/plugins/users-permissions/server/bootstrap/index.js

```diff
@@ -27,7 +27,14 @@ module.exports = async ({ strapi }) => {
   await getService('users-permissions').initialize();
 
   if (!strapi.config.get('plugin.users-permissions.jwtSecret')) {
-    const jwtSecret = uuid();
+    const jwtSecret = crypto.randomBytes(16).toString('base64');
+
+    if (process.env.NODE_ENV === 'production') {
+      throw new Error(
+        `[Users & Permissions] Missing jwtSecret. Please set jwtSecret in your config or set environment variable JWT_SECRET (ex: ${jwtSecret}).`
```

Comment:
I think it might not be clear to which config you are referring to and therefore I'd suggest to either:

- mention the (file)-path of the config explicitly
- same as above: link to the proper jwt config section in the documentation

What do you think?

Reply:
I agree, I'll modify it :) 
Regarding the link to documentation we can maybe wait the discussion with the doc team?